website/src/pages/initiatives.js

```diff
@@ -71,37 +106,34 @@ const Community = () => {
           </div>
         </section>
         <section id="meetings">
-          <h3 className={styles.communitySubtitle}>Our Meetings</h3>
-          <div className={styles.communityRow}>
-            <div className={styles.communityMeeting}>
+        <h3 className={styles.communitySubtitle}>Our Meetings</h3>
+           <div>
+            <div> 
               <h5 className={styles.meetingTitle}>Past Meetings</h5>
-              <List
-                height={500}
-                itemCount={pastMeetings.length}
-                itemSize={100}
-                width="90%"
-                itemData={{ isCompleted: true }}
-              >
-                {MeetingCard}
-              </List>
+                
+                  <Calendar className={styles.Calendar}
```

Comment:
The Styles currently fail in `Dark Mode` toggle and the I feel that color schemes and contrast can be improved. Small UI changes can be implemented to make it look even better.